index.html

```diff
@@ -32,12 +32,154 @@ <h1> Charlie
+    </div>
+  
+    <div class="projects-3">
+      <div class="projects-3_img">
+      <object  data="./svgfiles/Rectangle 2.svg" type="image/svg+xml" ></object>
+      </div>
+      <div class="project-3_name">
+        <h2>Project name goes here</h2>
+        <div class="project-1_name_lang">
+        <div class="langs">HTML/CSS</div>
+        <div class="langs-1">Ruby on Rails</div>
+        <div class="langs-2">JavaScript</div>
+        </div>
+        <button class="see-pro">See this project &rarr;</button>
+      </div>
+  </div>
+
+    <div class="projects-4">
+      <div class="projects-4_img">
+      <object  data="./svgfiles/Rectangle 6.svg" type="image/svg+xml" ></object>
+      </div>
+      <div class="project-4_name">
+        <h2>Project name goes here</h2>
+        <div class="project-1_name_lang">
+        <div class="langs">HTML/CSS</div>
+        <div class="langs-1">Ruby on Rails</div>
+        <div class="langs-2">JavaScript</div>
+        </div>
+        <button class="see-pro">See this project &rarr;</button>
+      </div>
+    </div>
+
+
+    <div class="projects-5">
+      <div class="projects-5_img">
+         <object  data="./svgfiles/Rectangle 4.svg" type="image/svg+xml" ></object>
+     </div>
+     <div class="project-5_name">
+        <h2>Project name goes here</h2>
+        <div class="project-1_name_lang">
+        <div class="langs">HTML/CSS</div>
+        <div class="langs-1">Ruby on Rails</div>
+        <div class="langs-2">JavaScript</div>
+        </div>
+        <button class="see-pro">See this project &rarr;</button>
+     </div>
+
+    </div>
+
+    <div class="projects-6">
+      <div class="projects-6_img">
+      <object  data="./svgfiles/Rectangle 5.svg" type="image/svg+xml" ></object>
+      </div>
+      <div class="project-6_name">
+        <h2>Project name goes here</h2>
+        <div class="project-1_name_lang">
+        <div class="langs">HTML/CSS</div>
+        <div class="langs-1">Ruby on Rails</div>
+        <div class="langs-2">JavaScript</div>
+        </div>
+        <button class="see-pro">See this project &rarr;</button>
+      </div>
+    </div>
+
+  </section>
+
+  <section class="Aboutme">
+    <div class="Aboutme-intro">
+    <div class="Aboutme--h"><h2 >About</h2></div>
+    <div class="Aboutme-para">
+     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
+      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+       quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p> 
+    </div>
+    <button class="cv">Get my CV  <object class="cvimg" data="./svgfiles/ic_download.svg" type="image/svg+xml" ></object></button>
+  </div>
+    <div class="Aboutme-grid">
+    <div class="Aboutme-lang">
+      <h3>Languages</h3>
+      <object class="lanimg" data="./svgfiles/ic_languages_mob.svg" type="image/svg+xml" ></object>
+      <ul>
+        <li>JavaScript(ES5/ES6)</li>
+        <li>Ruby</li>
+        <li>HTML</li>
```

Comment:
- [ ] I think it would be better also to align the content of the **About section** with the [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) to make it easier to create responsive layouts that can adapt to different screens sizes and devices. 😉